rustual-boy-cli/src/emulator.rs

```diff
@@ -124,13 +117,15 @@ impl Emulator {
 
             match self.mode {
                 Mode::Running => {
-                    let mut start_debugger = false;
+                    self.read_input_keys();
```

Comment:
Why was this changed? I specifically made sure `read_input_keys` was only happening once per frame as that can actually mess up logic in some games, since we're not latching the values properly in the game pad implementation. Also, since this is a PR about color spaces, it would've been nice to keep this separate.